components/helm-broker/internal/bundle/form.go

```diff
@@ -36,6 +37,11 @@ type FormMeta struct {
 	PlanUpdatable       *bool             `yaml:"planUpdatable"`
 }
 
+// DocsMeta contains data about bundle's docs fetched from docs/meta.yaml file
+type DocsMeta struct {
+	Docs []internal.BundleDocs `yaml:"docs"`
```

Comment:
you should have a `Validate` method which will ensure that there is no more that one entry

Reply:
added